Assets/MRTK/Core/Utilities/MixedRealitySceneContent.cs

```diff
@@ -0,0 +1,116 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using UnityEngine;
+using Microsoft.MixedReality.Toolkit.Utilities;
+
+#if UNITY_2017_2_OR_NEWER
+using System.Collections;
+using UnityEngine.XR;
+#else
+using UnityEngine.VR;
+#endif
+
+namespace Microsoft.MixedReality.Toolkit
```

Comment:
more of a nit, but this feels a little odd to be a "Core" component. I've tried (and definitely failed in some cases) to make `MonoBehaviour`s an SDK+ component only

Reply:
Unfortunately the MixedRealityInspectorUtility uses this class to get the scene ready, so it has to stay in core for now.